packages/webamp-modern-2/src/skin/BitmapFont.ts

```diff
@@ -2,7 +2,7 @@ import { num, px } from "../utils";
 import ImageManager from "./ImageManager";
 
 // TODO: Fill out the rest of this
```

Comment:
Should we just get rid of this comment now? I think the characters plausibly right to me now.

Reply:
Good point, getting rid of it now 👍